skelebot/systems/generators/dockerfile.py

```diff
@@ -55,9 +56,8 @@ def buildDockerfile(config):
     # Set the CMD to execute the primary job by default (if there is one)
     for job in config.jobs:
         if config.primaryJob == job.name:
-            ext = job.source.split(".")[1]
-            docker += "CMD /bin/bash -c '{command}/app/{job}'\n"
-            docker = docker.format(command=EXT_COMMAND[ext], job=job.source)
```

Comment:
is `EXT_COMMAND` still being used elsewhere in this file?

Reply:
Yup, Removed import